### ISSUE TITLE
[evaluation issue]: email not working

### ISSUE THREAD

**Issue by xavibou** [CAPTAIN-WHU/DOTA#26]
Hello,

I am trying to register to the DOTA evaluation server to be able to test my predictions, but I never received any confirmation link to my email. I registered under two different emails from educational institutions (one of them .edu). I also sent an email to dotawebsite3@gmail.com detailing my issues but never got a reply.

How should I proceed?

Thanks

**Comment by XWP-PPP** [comment]
Hello, I have encountered the same problem. Have you solved this problem？
Thank you

**Comment by xavibou** [comment]
No, I did not get any reply nor managed to get a confirmation link sent to my email
